collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -525,14 +525,18 @@ public void authRequest(Uri url, HashMap<String, String> doneSoFar) {
     }
 
     private class GoogleSheetsInstanceUploaderTask extends
-            GoogleSheetsAbstractUploader<Long, Integer, HashMap<String, String>> {
+            GoogleSheetsAbstractUploader {
 
         GoogleSheetsInstanceUploaderTask(GoogleAccountCredential credential) {
             HttpTransport transport = AndroidHttp.newCompatibleTransport();
             JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-            mService = new com.google.api.services.sheets.v4.Sheets.Builder(
+            mSheetsService = new com.google.api.services.sheets.v4.Sheets.Builder(
+                    transport, jsonFactory, credential)
+                    .setApplicationName("ODK-Collect-Sheets")
```

Comment:
I agree with you. I do think it makes more sense to keep only the application name in the User Agent Header. No, there wasn't any particular reason for naming it like this. I'll change both of them to "ODK-Collect". Does that sound ok?